FirebaseFunctions/Tests/Unit/FunctionsSerializerTests.swift

```diff
@@ -268,6 +366,6 @@ class FunctionsSerializerTests: XCTestCase {
 }
 
 /// Used to represent a type that cannot be encoded or decoded.
-private struct CustomObject {
+private class CustomObject {
```

Comment:
`NSArray` and `NSDictionary` implicitly wrap Swift value types in `__SwiftValue`.

Without this change, `XCTAssertEqual(typeName, "CustomObject")` in the collection tests results in `XCTAssertEqual failed: ("__SwiftValue") is not equal to ("CustomObject")`.